package.json

```diff
@@ -58,9 +58,8 @@
     "elasticsearchWithPlugins": "grunt esvm:withPlugins:keepalive",
     "lint": "grunt eslint:source",
     "lintroller": "grunt eslint:fixSource",
-    "makelogs": "makelogs",
-    "mocha": "mocha",
-    "mocha:debug": "mocha --debug-brk",
+    "makelogs": "echo 'use `node script/makelogs`' && false",
+    "mocha": "echo 'use `node script/mocha`' && false",
```

Comment:
Also: why `&& false`? In case scripts rely on these and you want exit code = 1?

npm is sooo noisy:

<img width="696" alt="screen shot 2017-05-09 at 10 11 23" src="https://cloud.githubusercontent.com/assets/5259/25843644/e6941f44-349f-11e7-8582-6ccad98e0d6d.png">